docs/source/tutorials/outliers.ipynb

```diff
@@ -476,50 +495,127 @@
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "740a8774",
+   "id": "40fed4ef",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Preprocess data\n",
+    "train_labels = np.array(train_data.dataset.targets)[train_data.indices]\n",
+    "train_labels = np.unique(train_labels, return_inverse=True)[1] # Zero index out train labels\n",
+    "test_labels = np.array(test_data.dataset.targets)[test_data.indices]\n",
+    "\n",
+    "scaler = preprocessing.StandardScaler().fit(train_feature_embeddings)\n",
+    "train_feature_embeddings_scaled = scaler.transform(train_feature_embeddings)\n",
+    "test_feature_embeddings_scaled = scaler.transform(test_feature_embeddings)\n",
+    "\n",
+    "# Split test data into 50/50 animal and non animal\n",
+    "animal_idxs = np.where(np.isin(test_labels, animal_classes))[0]\n",
+    "non_animal_idxs = np.where(np.isin(test_labels, non_animal_classes))[0]\n",
+    "test_idxs = np.random.choice(animal_idxs, len(non_animal_idxs), replace=False) + non_animal_idxs\n",
+    "test_idxs = np.hstack([np.random.choice(animal_idxs, len(non_animal_idxs), replace=False), non_animal_idxs])\n",
+    "test_labels = test_labels[test_idxs]\n",
+    "test_feature_embeddings_scaled = test_feature_embeddings_scaled[test_idxs]\n",
+    "\n",
+    "# Create model and use cross validation to generate held out pred-probs (may take a few minutes)\n",
+    "num_classes_indistribution = len(animal_classes)\n",
+    "model = BaggingClassifier(MLPClassifier(solver='adam', hidden_layer_sizes=(512, 64, num_classes_indistribution), random_state=1),random_state=1, n_jobs=-1)\n",
+    "\n",
+    "num_crossval_folds = 3  # for efficiency; values like 5 or 10 will generally work better\n",
+    "train_pred_probs = cross_val_predict(\n",
+    "    model,\n",
+    "    train_feature_embeddings_scaled,\n",
+    "    train_labels,\n",
+    "    cv=num_crossval_folds,\n",
+    "    method=\"predict_proba\",\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "03e3f7b7",
+   "metadata": {},
+   "source": [
+    "We can compute out-of-distribution predictions scores for the train and test datas using cleanlab's `OutOfDistribution` class."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "e110fc4b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "train_pred_labels = train_pred_probs.argmax(1)\n",
+    "accuracy = np.mean(train_pred_labels == train_labels)\n",
+    "print(f\"Model accuracy on held-out train_data {accuracy}\")\n",
+    "\n",
+    "ood = OutOfDistribution()\n",
+    "train_ood_predictions_scores = ood.fit_score(pred_probs=train_pred_probs, labels=train_labels)\n",
+    "\n",
+    "model = BaggingClassifier(MLPClassifier(solver='adam', hidden_layer_sizes=(512, 64, num_classes_indistribution), random_state=1),random_state=1, n_jobs=-1)\n",
```

Comment:
I don't think we need the `random_state=1` inside of the MLPClassfier?

From: 
https://scikit-learn.org/stable/modules/generated/sklearn.ensemble.BaggingClassifier.html
> random_state Controls the random resampling of the original dataset (sample wise and feature wise). If the base estimator accepts a random_state attribute, a different seed is generated for each instance in the ensemble.

Reply:
why not include: `early_stopping=True` in MLPClassifier?